common/djangoapps/xmodule_modifiers.py

```diff
@@ -200,7 +204,7 @@ def add_staff_markup(user, has_instructor_access, block, view, frag, context):
     Does nothing if module is a SequenceModule.
     """
     # TODO: make this more general, eg use an XModule attribute instead
```

Comment:
Yes, they still apply. Ideally, the staff info would be an XBlock aside that only applied in the right spots.